api/.env-example

```diff
@@ -5,10 +5,10 @@ PORT=3001
 # docker run --name mysql -e MYSQL_ROOT_PASSWORD=mysql-root-password -e MYSQL_DATABASE=my-database -d -p 3306:3306 mysql:latest
 DB_CLIENT=mysql2
 DB_HOST=localhost
-DB_PORT=3306
+DB_PORT=3308
 DB_USER=root
-DB_PASSWORD=mysql-root-password
-DB_DATABASE_NAME=meal-sharing
+DB_PASSWORD=my-secret-pw 
+DB_DATABASE_NAME=restaurant
 
 # The database you use for local development most likely doesn't use SSL (encryption)
 # but the one you deploy on Render.com does.
```

Comment:
This file was supposed to be renamed from `.env-example` to `.env`. then `.env` is listed in your `.gitignore` here:
`api/.gitignore`

This means your code repository never commits your passwords into the version control software `git`. Isn't that clever.

Here it seems you have shared your passwords with the world

Reply:
I made changes